add delete text

p-galligan opened the pull request:
Adding the delete section to the Readme. Fixes #12 

As with our other issue, we're not quite sure what else there is to this.

Commit:
add delete text

@daremiyo here's the deletion section.

As with our other issue, we're not quite sure what else there is to this.

helrond reviewed (changes requested):
This is a really useful distinction, I think (and one that I hadn't really thought about). I wonder if we can build this out with a bit more technical detail about what is required in each case? The first would require a DELETE request against an endpoint, while the second requires a POST with some JSON as a payload? I think that will help us see where the overlaps are.

p-galligan committed:
adding additional delete information

adding additional deletion information and adding a method to identify and delete orphaned objects

p-galligan commented:
@helrond and @HaSistrunk added some more specific data and also added some text about deleting orphan containers, and idea I just had this morning.

HaSistrunk committed:
tweak langauge to standardize

helrond reviewed (approved):
I'm not convinced we need a `delete_orphans()` method (or that this library needs to cover delete requests to existing endpoints since I'm not sure what the gain is there) but I'd argue for leaving this as-is for now because it's a really comprehensive scoping of the problem which we can refine as we start to write code.
